Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "sn_consensus"
-version = "3.3.2"
+version = "3.4.0"
```

Comment:
Don't bump the version number. We have a robot that manages the release process for us. That involves:
1. bumping the version numbers
2. creating tags + a github release
3. publishing the crate to crates.io

It can trip up the bot if you bump numbers yourself.

Reply:
Thanks for the explanation. It's reverted.